.eslintrc.js

```diff
@@ -0,0 +1,26 @@
+module.exports = {
```

Comment:
@notmessenger How does the ocd sort import declarations differ from what was established? I've always used that rule, and sorted my imports that way even before the rule existed? Who established the alternative to which you refer and what is that alternative?